src/mapping/recurrences/attractor_mapping_recurrences.jl

```diff
@@ -174,7 +174,20 @@ end
 
 extract_attractors(m::AttractorsViaRecurrences) = m.bsn_nfo.attractors
 
-iterations_to_converge(m::AttractorsViaRecurrences) =  m.bsn_nfo.safety_counter
+function iterations_to_converge(m::AttractorsViaRecurrences)
+    i = m.bsn_nfo.safety_counter
+    kw = m.kwargs
+    if m.bsn_nfo.return_code == :new_att
+        # in this scenario we have an addition amount of iterations that is
+        # equal to the recurrences steps. We subtract this for more correct
+        # estimation of convergence time that will be closer to the convergence
+        # time of neighboring grid cells.
+        x = get(kw, :consecutive_recurrences, 100) + get(kw, :attractor_locate_steps, 1000)
+    else
+        x = get(kw, :consecutive_attractor_steps, 2)
```

Comment:
elseif m.bsn_nfo.return_code == :bas_hit
   x = 0

Reply:
I think we should actually return `NaN` in the `:bas_hit` case. because in this scenario the reported convergence time can be wildly inaccurate. We really can't even quantify how inaccurate it will be.!